packages/core/src/wallet/IndyWallet.ts

```diff
@@ -80,43 +68,40 @@ export class IndyWallet implements Wallet {
 
     // Open wallet, creating if it doesn't exist yet
     try {
-      await this.open(walletConfig, walletCredentials)
+      await this.open(walletConfig)
     } catch (error) {
       // If the wallet does not exist yet, create it and try to open again
       if (error instanceof WalletNotFoundError) {
-        await this.create(walletConfig, walletCredentials)
-        await this.open(walletConfig, walletCredentials)
+        await this.create(walletConfig)
+        await this.open(walletConfig)
       } else {
         throw error
```

Comment:
Should this not be an `AriesFrameworkError` or `IndySdkError`?

Reply:
`this.open` already handles this, so we'll always get an aries framework error here